### PR TITLE
fix: missing affine directory in publish script

### PR DESCRIPTION
**Pull request by L-Sun**

### CODE REVIEW

**Comment by L-Sun** [comment]
* **#7926** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7926?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @L-Sun and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: scripts/publish.sh [hunk 1]
```diff
@@ -18,21 +18,21 @@ packages=(
...
 for package in "${packages[@]}"
 do
   cd "packages/$package"
 
-  if [ "$CANARY" = "true" ]; then
-    pnpm publish --no-git-checks --tag canary
-  else
-    pnpm publish
-  fi
+  # if [ "$CANARY" = "true" ]; then
```
Comment (Saul-Mirone) [inline scripts/publish.sh:29]: Why comment this?

Comment (L-Sun) [reply scripts/publish.sh]: Sorry about that. It was for debugging purposes. I have uncommented it.

**Review by Saul-Mirone — APPROVED** [review]
Nice catch 👍

File: scripts/publish.sh (hunk 1 above)
Comment (Saul-Mirone) [reply scripts/publish.sh]: NP

**Pull request merged**